docs/layout/index.hbs

```diff
@@ -4,15 +4,238 @@ description: Layout your app according to your needs using the AXA Web Toolkit.
+    slug: drawer-component
+  - title: Reponsive utilities
+    slug: responsive-utilities
 ---
 
 {{#> page}}
   {{#*inline "content"}}
 
-    <div class="alert alert-info" role="alert" style="margin: 50px 0;">
-      <p><strong>Heads up!</strong> This page is being rewritten
-      in order to layouting with the AXA Web Toolkit
-      as easy as possible. Come back again in a couple of days!</p>
+    <h1 class="docs-h2">{{title}}</h1>
+
+    <p class="lead">The Web Toolkit provides several ways
+    for laying out your web application according to your user's needs.
+    This contains wrapping containers, a Bootstrap-based grid system,
+    a powerful Drawer component, and responsive utility classes.</p>
+
+    {{> table-of-contents data=toc}}
+
+    <h2 id="containers" class="docs-h3">Containers</h2>
+
+    <p>As in Bootstrap, you have two containers available. The default
+    one should always be the ordinary <code>container</code>, which restricts
+    your content to a certain width depending on your viewport.
+    This behavior is perfect for all sort of informational websites,
+    because it keeps things readable.</p>
+
+    <div class="docs-example">
+      <div class="docs-example-content">
+        <div class="docs-layout-example">
+          <div class="docs-layout-example-container">
+            <div class="docs-layout-example-header"></div>
+            <div class="docs-layout-example-sidebar"></div>
+            <div class="docs-layout-example-body"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    {{#code-snippet "html"}}
+      <div class="container">
+        <!-- content here -->
+      </div>
+    {{/code-snippet}}
+
+    <p>The alternative is <code>container-fluid</code>. It doesn't
+    restrict to any specific width, keeping the whole browser estate
+    for your content. You should only use this when building complex
+    transactional websites, that need all the browser space available.</p>
+
+    <div class="docs-example">
+      <div class="docs-example-content">
+        <div class="docs-layout-example">
+          <div class="docs-layout-example-container-fluid">
+            <div class="docs-layout-example-header"></div>
+            <div class="docs-layout-example-sidebar"></div>
+            <div class="docs-layout-example-body"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    {{#code-snippet "html"}}
+      <div class="container-fluid">
+        <!-- content here -->
+      </div>
+    {{/code-snippet}}
+
+    <h2 id="grid" class="docs-h3">Grid</h2>
+
+    <p>The AXA Web Toolkit uses the 12-column Bootstrap grid. Whenever
+    you use the grid to split some content into multiple columns, make
+    sure you place it inside a `container` or `container-fluid`.</p>
+
+    <p>You can find the full documentation of the grid in the official
+    <a href="{{gridDocs}}">Bootstrap grid docs</a>.</p>
+
+    <h2 id="drawer-component" class="docs-h3">Drawer component</h2>
```

Comment:
Naming might be a problem. I named it like this because it is a container that allows you to put different things in it. You don't necessarily need to put a menu in it. You can only have a search in there. Or a couple of buttons.